### PR TITLE
refactor: update to yeoman-generator@1.0

### PR DESCRIPTION
**Pull request by iamstarkov**

### CODE REVIEW

**Comment by iamstarkov** [comment]
@SBoudrias hi, can you help me out with migration to yeoman-generator@1.x?

File: generators/app/index.js [hunk 1]
```diff
@@ -10,32 +10,30 @@ function sortByKeys(a, b) {
   return travisConfigKeys.indexOf(a) < travisConfigKeys.indexOf(b) ? -1 : 1;
 }
 
-module.exports = yeoman.Base.extend({
-  constructor: function () {
-    yeoman.Base.apply(this, arguments);
+module.exports = class extends Generator {
+  constructor(...args) {
```
Comment (SBoudrias) [inline generators/app/index.js]: That syntax is not supported in Node 4 unfortunately.

**Comment by SBoudrias** [comment]
Working now ;)

**Comment by iamstarkov** [comment]
hurray

**Pull request merged**